src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx

```diff
@@ -9,6 +9,46 @@ import { showRelinquishedEffectiveDate } from '../../utils/helpers';
 
 export default class BenefitsRelinquishmentFields extends React.Component {
   render() {
+    const dateFields = (
+      <div>
+        <DateInput required={showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)}
+            allowFutureDates
+            errorMessage="Date cannot be earlier than 2 years ago"
+            validation={validateIfDirtyDateObj(this.props.data.benefitsRelinquishedDate, isValidRelinquishedDate)}
+            label="Effective date"
+            name="benefitsRelinquishedDate"
+            day={this.props.data.benefitsRelinquishedDate.day}
+            month={this.props.data.benefitsRelinquishedDate.month}
+            year={this.props.data.benefitsRelinquishedDate.year}
+            onValueChange={(update) => {this.props.onStateChange('benefitsRelinquishedDate', update);}}/>
+        <div>
+          <ul>
+            <li>Use today’s date unless you aren’t going to use your Post 9/11 GI Bill benefits until later.</li>
+            <li>If you pick a future date, you can’t get benefits until then.</li>
+            <li>If your classes started less than 2 years ago, enter the date they began.</li>
+          </ul>
+        </div>
+      </div>
+    );
+
+    // nest DateInputs inside radio buttons
+    const options = relinquishableBenefits.map((d) => {
```

Comment:
Sure, that's fine.  I noticed we don't have a ton of consistency in this arena.  Maybe we should add an eslint rule...